contracts/proposals/ccip016-missed-payouts.clar

```diff
@@ -0,0 +1,421 @@
+;; TRAITS
+
+(impl-trait .proposal-trait.proposal-trait)
+(impl-trait .ccip015-trait.ccip015-trait)
+
+;; ERRORS
+
+(define-constant ERR_PANIC (err u16000))
+(define-constant ERR_SAVING_VOTE (err u16001))
+(define-constant ERR_VOTED_ALREADY (err u16002))
+(define-constant ERR_NOTHING_STACKED (err u16003))
+(define-constant ERR_USER_NOT_FOUND (err u16004))
+(define-constant ERR_PROPOSAL_NOT_ACTIVE (err u16005))
+(define-constant ERR_PROPOSAL_STILL_ACTIVE (err u16006))
+(define-constant ERR_VOTE_FAILED (err u16007))
+
+;; CONSTANTS
+
+(define-constant SELF (as-contract tx-sender))
+(define-constant CCIP_016 {
+  name: "Refund Incorrect CCD007 Payouts",
+  link: "https://github.com/citycoins/governance/blob/feat/add-ccip-016/ccips/ccip-016/ccip-016-refund-incorrect-ccd007-payouts.md",
+  hash: "",
```

Comment:
Noting that the [CCIP-016 governance text](https://github.com/citycoins/governance/blob/feat/add-ccip-016/ccips/ccip-016/ccip-016-refund-incorrect-ccd007-payouts.md) needs to be updated and hash can be added here.